lib/external_service_new/zendesk_dispatcher.rb

```diff
@@ -0,0 +1,30 @@
+require "oj"
+require "rest_client"
+require "external_service_new/rest_client_dispatcher"
+
+module ExternalServiceNew
+  class ZendeskDispatcher < RestClientDispatcher
+		attr_reader :site_url
```

Comment:
Unless its git tripping, :P , Noticing your files might be indented with 4 spaces or more. We want 2 spaces indent (no tabs, or course). Please verify all your files.